src/App.js

```diff
@@ -1,20 +1,27 @@
 import React, { Component } from 'react'
-import movieData from './movieData'
 import Movies from './Movies'
 import SingleMovie from './SingleMovie'
+import { fetchData } from './APIcalls'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: { movieData },
+      movies: '',
```

Comment:
Movies is going to be an array eventaully, right? Then it's default value should be an array too!